QuoteGeneration/quote_wrapper/quote/id_enclave/id_enclave.cpp

```diff
@@ -288,3 +308,201 @@ sgx_status_t ide_get_pce_encrypt_key(
 
     return sgx_status;
 }
+
+sgx_status_t ide_get_ppid_encrypt_key(
+        const sgx_target_info_t* p_pce_target_info,
+        sgx_report_t* p_ide_report,
+        uint8_t crypto_suite,
+        uint16_t cert_key_type,
+        uint32_t key_size,
+        uint8_t* p_public_key)
+{
+    ref_rsa_params_t g_rsa_key = { 0 };  // The private key used to encrypt the PPID.  Only used for PPID_CEARTEXT Cert_Data_Type
+    return get_pce_encrypt_decrypt_key(p_pce_target_info, p_ide_report, crypto_suite, cert_key_type, key_size, p_public_key, &g_rsa_key);
+}
+
+sgx_status_t ide_decrypt_ppid(
```

Comment:
Copy of https://github.com/raoulstrackx/SGXDataCenterAttestationPrimitives/blob/e0a57b5d25606f24d7d320ac50df6514117f3a9b/QuoteGeneration/quote_wrapper/quote/enclave/quoting_enclave_3.cpp#L1355